src/main/java/spotify/api/enums/EntityType.java

```diff
@@ -1,11 +1,28 @@
 package spotify.api.enums;
 
+
+/**
+ * This class contains and sets the suitable values for the entity type. The entity
```

Comment:
Replace `This class contains and sets the suitable values for the entity type` with `This class contains all possible entity types`